JSQMessagesDemo/DemoMessagesViewController.m

```diff
@@ -111,7 +115,17 @@ - (void)viewDidAppear:(BOOL)animated
     self.collectionView.collectionViewLayout.springinessEnabled = [NSUserDefaults springinessSetting];
 }
 
+- (void)preferredContentSizeChanged:(NSNotification *)notification
+{
+    [self.collectionView layoutSubviews];
```

Comment:
Also, `JSQMessagesViewController`should be listening and respond to this notification... (so clients don't have to add this boilerplate)